docs/MPQEditor.md

```diff
@@ -0,0 +1,63 @@
+# MixedPrecisionQantizationEditor (MPQEditor for short)
+
+## About
+
+MPQEditor povides easy way of creation/editing MPQ json files, suitable for Quantization step in CfgEditor. Also it gives the ability to select the most sensitive to quantization layers using `visq` data. 
+
+## How to use
+
+Currently MPQEditor is integrated into EXPLORER view by context menu item "Create MPQ json".
+
+In order to use it:
+1) call context menu for a `.circle model` in EXPLORER view
+2) call "Create MPQ json"
+3) give a name to MPQ json
+4) edit MPQ json using MPQEditor
+5) all layers from 'layers' section in json are shown in grid-table, so that their options for quantization can be edited using their respective dropdowns
+6) to make 'model-graph-view' more informative the user may use 'visq' information, it shows which layers are more sensitive to quantization than others
+
+<img src="./images/MPQ_UI.png"/>
+
+## Implementation notes 
+By default `model-graph-view` is shown in `selector` mode to provide the way to select multiple nodes.
+When the user loads `visq` file, we need a way for fast resetting `__viewMode` of `index.html`, that is why new message `changemode` was introduced.
+Currently `viewMode.visq` can not be used, because it supports only single selection and for selected item `properties` panel is called.
+We need such a mode, that has the ability of showing `visq` colormaps along with the ability to select multiple nodes.
+That is why new `viewMode.visqselector` mode was introduced.
+
+## FAQ
```

Comment:
I don't know below questions should be in FAQ.
It was my question to understand MPQEditor feature.
I think explanation be better in introduction.

Reply:
Ok.